app/assets/javascripts/modules/track-links.js

```diff
@@ -5,10 +5,10 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   'use strict'
 
   Modules.TrackLinks = function () {
```

Comment:
With the conversion to init this will be broken up into two functions

This is the constructor:

```js
Module.TrackLinks = function (element) {
  // set this variables so probably:
  // this.category = element.getAttribute('data-track-category')
  // this.action = element.getAttribute('data-track-action')
  // this.links = element.getElementsByTagName('a')
}
```

Then you'll have the init function

```js
Modules.TrackLinks.prototype.init = function () { 
  // do the actual work e.g.
  // if (!this.category) return

  // for (var i = 0; i < this.links.length; i++) {
  // etc
}
```